src/simple_edg.rs

```diff
@@ -0,0 +1,158 @@
+/// Defines a hardcoded EDG.
+/// This allows us to easily create EDGs for testing and other purposes. It works well
+/// together with the `edg_assert` macro.
+///
+/// # Example A
+/// A simple EDG with 4 vertices A, B, C, D:
+/// ```
+/// simple_edg![
+///     A => -> {B, C} -> {D};
+///     B => ;
+///     C => -> {B} .> D;
+///     D => -> {};
+/// ];
+///
+/// edg_assert!(A, TRUE);
+/// edg_assert!(B, FALSE);
+/// ```
+/// We list the edges of each vertex, separating each vertex using a semicolon.
+/// The syntax `-> {V, ..., W}` defines a hyper-edge with targets `V`,...,`W`, and the syntax
+/// `.> P` defines a negation edge with target `P`. Declare multiple edges by writing them
+/// after each other (as seen for vertex `A` and `C` above), but note that hyper-edges must
+/// precede negation edges. Additionally, all targets must also be declared in the first line
+/// of the macro, and the vertex declarations and the edge declarations are separated with `::`.
```

Comment:
The note about the `::` syntax needs to be removed since we no longer have explicit vertex enumeration